plugins/lookup/etcd3.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+#
+# (c) 2020, SCC France, Eric Belhomme <ebelhomme@fr.scc.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    author:
+    - Eric Belhomme <ebelhomme@fr.scc.com>
+    lookup: etcd3
+    short_description: Get key values from etcd3 server
+    description:
+    - Retrieves key values and/or key prefixes from etcd3 using gRPC API.
+
+    options:
+        _terms:
+            description:
+            - The list of keys to look up on the etcd3 server.
+            type: list
+            elements: str
+            required: True
+        prefix:
+            description:
+            - Look for key or prefix key.
+            type: bool
+        host:
+            description:
+            - etcd3 listening client host.
+            default: '127.0.0.1'
+            type: str
+        port:
+            description:
+            - etcd3 listening client port.
+            default: 2379
+            type: int
+        ca_cert:
+            description:
+            - etcd3 CA authority.
+            env:
+            - name: ETCDCTL_CACERT
+            type: str
+        cert_cert:
+            description:
+            - etcd3 client certificate.
+            env:
+            - name: ETCDCTL_CERT
+            type: str
+        cert_key:
+            description:
+            - etcd3 client private key.
+            env:
+            - name: ETCDCTL_KEY
+            type: str
+        timeout:
+            description:
+            - Client timeout.
+            default: 60
+            env:
+            - name: ETCDCTL_DIAL_TIMEOUT
+            type: int
+        user:
+            description:
+            - Authentified user name.
+            env:
+            - name: ETCDCTL_USER
+            type: str
+        password:
+            description:
+            - Authentified user password.
+            env:
+            - name: ETCDCTL_PASSWORD
+            type: str
+        endpoints:
+            description:
+            - Counterpart of ETCDCTL_ENDPOINTS enviroment variable.
+              Specify the etcd3 connection with and URL form or <host>:<port> form
```

Comment:
```suggestion
              Specify the etcd3 connection with and URL form or C(<host>:<port>) form
```

Reply:
just for my information, what "C() actually does ?

Reply:
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation